elements/urth-core-dataframe/urth-core-dataframe.html

```diff
@@ -272,9 +281,28 @@
          */
         onModelValueChange: function(newVal){
             this._debug( "urth-core-dataframe onModelValueChange", newVal );
+            newVal = this._deserializeColumnTypes(newVal);
             this._setValue(newVal || {data:[]});
         },
 
+        /*
+         * Returns the deserialized DataFrame reconstructed via the column_types
+         * @param df - the serialized DataFrame
+         * @return the reconstructed DataFrame
+         */
+        _deserializeColumnTypes: function(df) {
+            for (var i=0; i<df.column_types.length; i++) {
+                //if this column is of type date convert the string content to a date object
+                if(df.column_types[i] === "datetime64[ns]" || df.column_types[i] === "Date"
```

Comment:
are other values possible within the square brackets?  Could replace the entire condition with a regular expression, something like `df.column_types[i].match('^datetime64[.*]|Date|TimestampType$')`